tests/unit/drivers/test_rankingevaluation_driver.py

```diff
@@ -80,37 +89,45 @@ def req(self) -> 'jina_pb2.Request':
         return self.eval_request
 
 
-def test_evaluate_driver_matches_in_chunks():
+@pytest.fixture
+def simple_chunk_evaluate_driver():
+    return SimpleChunkEvaluateDriver()
+
+
+@pytest.fixture
+def eval_request():
+    num_docs = 10
+    num_matches = 1
+
+    def add_matches(doc: jina_pb2.Document):
+        for idx in range(num_matches):
+            match = doc.matches.add()
+            match.tags['id'] = idx
+
+    req = jina_pb2.Request.IndexRequest()
+    for idx in range(num_docs):
+        doc = req.docs.add()
+        gt = req.groundtruths.add()
+        chunk_doc = doc.chunks.add()
+        chunk_gt = gt.chunks.add()
+        chunk_doc.granularity = 1
+        chunk_gt.granularity = 1
```

Comment:
These lines are common in many tests

Reply:
I do not mind having `tests` self contained.